include/parlay/internal/integer_sort.h

```diff
@@ -219,7 +219,7 @@ sequence<size_t> integer_sort_r(slice<InIterator, InIterator> In,
       // uninitialized_relocate_n, which can memcpy multiple elements at a time
       // to save on performing every copy individually.
       if constexpr (std::is_same_v<assignment_tag, uninitialized_relocate_tag>) {
-        uninitialized_relocate_n(Out.begin(), In.begin(), Out.size());
+        parlay::uninitialized_relocate(In.begin(), In.end(), Out.begin());
```

Comment:
Throughout: I guess `In.size() == Out.size()`? which is also known as `n`? You switch from `Out.size()` on the left to `In.end()` on the right, so they better be the same size.